src/pages/DoctorDetail.tsx

```diff
@@ -0,0 +1,73 @@
+import { useParams } from "react-router-dom";
+import { MdVerified } from "react-icons/md";
+import { IoIosInformationCircleOutline } from "react-icons/io";
+import { fetchDoctors } from "../components/services/patientService";
+import { specialityImages } from "../assets/assets";
+import useSWR from "swr";
+import { Doctor } from "./ModalDoctorPage";
+
+export function DoctorDetailsFetcher() {
+  const { doctorId } = useParams();
+  const { data, error, isLoading } = useSWR("doctors", fetchDoctors);
+
+  console.log("Doctor ID from URL:", doctorId);
+
+  if (isLoading) return <div>Loading doctor details...</div>;
+  if (error) return <div>An error occurred: {error.message}</div>;
+
+  const doctor = data?.find((doc: { id: string }) => doc.id === doctorId);
+
+  if (!doctor) return <div>No doctor found.</div>;
+
+  return <DoctorDetail doctor={doctor} />;
+}
+
+export function DoctorDetail({ doctor }: { doctor: Doctor }) {
+  return (
+    <>
+      {doctor ? (
+        <div className="flex gap-8 p-8 max-w-screen-lg mx-auto">
```

Comment:
Może znajdą się lepsze znaczniki, np. section? article?

div to wyjscie absolutnie ostateczne zawsze